Tribler/Core/SessionConfig.py

```diff
@@ -185,6 +185,27 @@ def get_permid_keypair_filename(self):
             self.set_permid_keypair_filename(file_name)
         return file_name
 
+    def set_multichain_permid_keypair_filename(self, keypairfilename):
+        """ Set the filename containing the Elliptic Curve keypair to use for
+        PermID-based authentication for multichain in this Session.
+
+        Note: if a Session is started with a SessionStartupConfig that
+        points to an existing state dir and that state dir contains a saved
+        keypair, that keypair will be used unless a different keypair is
+        explicitly configured via this method.
+        """
+        self.sessconfig.set(u'general', u'eckeypairfilenamemultichain', keypairfilename)
```

Comment:
Minor detail: I think it would be a bit nicer to use `ec_keypair_filename_multichain` as section key name.